### PR TITLE
Feat default minimum coordinates

### PR DESCRIPTION
**Pull request by IvarStefansson**
## Proposed changes

Provide default coordinates for "xmin" etc. when creating a box-shaped domain. For convenience in the very common case that xmin=ymin(=zmin)=0. A full purging of explicit declarations of "xmin": 0 etc. was not performed. I think this can wait for the test overhaul.

I suggest @jhabriel does initial review.

## Types of changes

What types of changes does this PR introduce to PorePy?
_Put an `x` in the boxes that apply._

- [ ] Minor change (e.g., dependency bumps, broken links).
- [ ] Bugfix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Testing (contribution related to testing of existing or new functionality).
- [ ] Documentation (contribution related to adding, improving, or fixing documentation).
- [ ] Maintenance (e.g., improve logic and performance, remove obsolete code).
- [ ] Other:

## Checklist

_Put an `x` in the boxes that apply or explain briefly why the box is not relevant._

- [x] The documentation is up-to-date.
- [x] Static typing is included in the update.
- [x] This PR does not duplicate existing functionality.
- [x] The update is covered by the test suite (including tests added in the PR).
- [ ] If new skipped tests have been introduced in this PR, `pytest` was run with the `--run-skipped` flag.

### CODE REVIEW

File: src/porepy/models/constitutive_laws.py [hunk 1]
```diff
@@ -747,7 +747,7 @@ class CubicLawPermeability(ConstantPermeability):
 
     equation_system: pp.ad.EquationSystem
     """EquationSystem object for the current model. Normally defined in a mixin class
```
Comment (OmarDuran) [inline src/porepy/models/constitutive_laws.py:749]:
```suggestion
    """EquationSystem object for the current model. Solution strategies are normally
```

File: src/porepy/models/constitutive_laws.py [hunk 2]
```diff
@@ -747,7 +747,7 @@ class CubicLawPermeability(ConstantPermeability):
...
-    defining the solution strategy.
+    definin the solution strategy.
```
Comment (OmarDuran) [inline src/porepy/models/constitutive_laws.py:750]:
```suggestion
     defined in a mixin class.
```

**Review by OmarDuran — CHANGES_REQUESTED** [review]
Many thanks. I have left one suggestion.

**Comment by IvarStefansson** [comment]
> Many thanks. I have left one suggestion.

Thanks! I have accepted your suggestions. Please approve the PR if you're happy (now it's stuck on "change requested").

**Review by OmarDuran — APPROVED** [review]

**Pull request merged**